include/libtorrent/settings_pack.hpp

```diff
@@ -876,6 +876,9 @@ namespace aux {
 			// any). This is only supported by SOCKS5 and HTTP.
 			proxy_hostnames,
 
+			// if true, listening on the defined port will be enabled while using proxy
+			listen_on_proxy,
```

Comment:
this enum value has to be added at the end, to prevent existing constants to change value.
I'm not super happy about this name. all other proxy settings have the `proxy_` prefix, maybe something like `proxy_accept_incoming`?

Reply:
Thx! fixed in [91081ce](https://github.com/arvidn/libtorrent/pull/7726/commits/91081ce7eb5eff128df3f8fe1f8294f95c1675aa)